main.cc

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
```

Comment:
Arrange includes alphabetically, see our C++ guidelines. List all C-includes before C++ includes

Reply:
List c-includes first, so `cmath` goes above `array` and `iostream`